### PR TITLE
Add stub files

### PR DESCRIPTION
**Pull request by yotamN**
I'm still struggling to compile Frida so I haven't tested it yet but it should be complete

### CODE REVIEW

**Comment by yotamN** [comment]
I've finally managed to compile Frida successfully so it can be merged now

File: frida/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,225 @@
+from typing import Any, Callable, Dict, List, Literal, Optional, Tuple, Union
+
+from . import core
+
+__version__: str
```
Comment (oleavr) [inline frida/__init__.pyi:5]: This doesn't seem to follow PEP-8 wrt. double blank separator lines. Or are the conventions different from .pyi? (I'm new to this 😄)

**Review by oleavr — COMMENTED** [review]
Thanks, this is awesome! 🙌

File: frida/__init__.pyi (hunk 1 above)
Comment (yotamN) [reply frida/__init__.pyi:5]: The format is a bit weird but I run it through Black so it should be correct. Anyway I'm a bit conflicted if it's worth merging it if the Python 3 only version would be merged soon. It has type hinting in the code itself which is a lot more pleasant to maintain.

Comment (oleavr) [reply frida/__init__.pyi:5]: Ahh good point. Let's merge the Python 3 only version instead then!

**Pull request closed without merging**